src/messaging/tests/BUILD.gn

```diff
@@ -49,7 +48,6 @@ chip_test_suite("tests") {
   tests = [
     "TestChannel",
     "TestExchangeMgr",
-    "TestMessageCounterSyncMgr",
```

Comment:
Oh, and @kghost what is the plan for re-enabling this test?  I figured https://github.com/project-chip/connectedhomeip/pull/5389 would do it, but it doesn't seem to?

Reply:
#5389 will enable it.

https://github.com/project-chip/connectedhomeip/pull/5389/files#diff-f3c5e54fca741772049ea3c36beff3eb0f304065a2dbaa381e1ed3ed79de21ad

The file will be rename and moved to src/protocols/message_counter/tests/TestMessageCounterManager.cpp